src/Azure.Functions.Sdk/WebJobsReference.FromModule.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using Mono.Cecil;
+using NuGet.Common;
+
+namespace Azure.Functions.Sdk;
+
+public partial class WebJobsReference
+{
+    private const string ExtensionsBinaryDirectoryPath = $@"./{Constants.ExtensionsOutputFolder}";
+    private const string WebJobsStartupAttributeType = "Microsoft.Azure.WebJobs.Hosting.WebJobsStartupAttribute";
+
+    /// <summary>
+    /// Gets the root path of the Azure Functions SDK module.
+    /// </summary>
+    public static IEnumerable<WebJobsReference> FromModule(string path, ILogger? logger = null)
+    {
+        Throw.IfNullOrEmpty(path);
+        FunctionsAssemblyResolver resolver = new();
+        ReaderParameters readerParameters = new() { AssemblyResolver = resolver };
+        AssemblyDefinition assembly = AssemblyDefinition.ReadAssembly(path, readerParameters);
```

Comment:
Same comment about disposing, applies to all ReadAssembly usage

Reply:
Class is removed for now.